src/index.ts

```diff
@@ -83,9 +84,25 @@ class ClientApp {
      *
      * clientApp.users.someMethod(...);
      * ```
+     *
+     * @deprecated Since 2.3.0.  @see [directory](#directory) for replacements
```

Comment:
We decided to deprecate the users module and api and replace it with the directory api since it better describes/encapsulates the sdk methods of the module.  We'll remove this in 3.0.0